functions/ts/kubeval/forked/github.com/instrumenta/openapi2jsonschema/command.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
```

Comment:
we should have third-party code in a `thirdparty` directory. You may want to separate committing the pristine copy of the thirdparty in a separate PR and include information such as git repo/commit.

Reply:
Created https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/240